modulelogger/examples_test.go

```diff
@@ -0,0 +1,127 @@
+package modulelogger
```

Comment:
can you call the package logging? calling it modulelogger is kind of just an implementation detail of the logging facility.